apps/api/src/app/message-queue/message-queue.service.ts

```diff
@@ -137,6 +142,51 @@ export class MessageQueueService implements OnModuleInit, OnModuleDestroy {
 
       await sendMail(from, to, subject, mailTemplate);
     }
+    else if (json_message.run_status === 'partial_success') {
+      const result = await this.prismaService.trial.updateMany({
+        where: {
+          trial_internal_id: {
+            in: trialInternalIds
+          },
+          trial_status: {
+            in: [TrialStatusEnum.PENDING, TrialStatusEnum.ERROR]
+          },
+        },
+        data: {
+          trial_status: TrialStatusEnum.MATCHED
+        }
+      });
+      const result2 = await this.prismaService.trial.updateMany({
+        where: {
+          trial_internal_id: {
+            in: failedTrialInternalIds
+          },
+          trial_status: {
+            in: [TrialStatusEnum.PENDING,TrialStatusEnum.MATCHED]
+            }
```

Comment:
when they are sent to matchminer they should all the PENDING, so I don't think MATCHED is needed here

Reply:
Yes, I think you are right. I was thinking something else. will fix